xbmc/filesystem/NFSFile.cpp

```diff
@@ -907,9 +959,14 @@ bool CNFSFile::OpenForWrite(const CURL& url, bool bOverWrite)
   {
     CLog::Log(LOGWARNING, "FileNFS::OpenForWrite() called with overwriting enabled! - {}",
               filename);
-    //create file with proper permissions
-    ret = nfs_creat(m_pNfsContext, filename.c_str(), S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH, &m_pFileHandle);
+    //create file with proper permissions.
+    //In NFS V4, the libnfs requires the flags O_EXCL to transmit the mode in the call
```

Comment:
These comments aren't needed.

Reply:
I have summarised the comment to make it clear that the flag is mandatory and thus avoid its removal.